service/resource/configmap/desired_test.go

```diff
@@ -241,6 +242,73 @@ func Test_Resource_ConfigMap_GetDesiredState(t *testing.T) {
 			},
 			expectedErrorHandler: nil,
 		},
+
+		// Test that if the configmap exists, with a service already,
```

Comment:
This is the actual new functionality - tl;dr - if a service is removed, we remove it from the prometheus configuration.

Reply:
I would like to see test numbers here as well.